apps/condo/lang/en/en.json

```diff
@@ -2347,7 +2350,7 @@
   "tasks.MeterReadingsImportTask.progress.title": "Import meter readings",
   "tasks.MeterReadingsImportTask.progress.description.preparing": "Preparing to import meter readings",
   "tasks.MeterReadingsImportTask.progress.description.processing": "Meter readings processed: {imported} of {total}",
-  "tasks.MeterReadingsImportTask.progress.description.completed": "File has been created",
+  "tasks.MeterReadingsImportTask.progress.description.completed": "File has been created, please refresh the page",
```

Comment:
why does user need to update pages? in other imports we refetch data after import completed

Reply:
temporary solution because of meters not refetching